packages/core/src/createGetBalances.ts

```diff
@@ -44,22 +46,40 @@ export const createGetBalances = ({ send }: Provider) =>
      * - `INVALID_THRESHOLD`: Invalid `threshold`
      * - Fetch error
      */
-    (addresses: ReadonlyArray<Hash>, threshold: number, callback?: Callback<Balances>): Promise<Balances> =>
-        Promise.resolve(
+    (
+        addresses: ReadonlyArray<Hash>,
+        threshold: number,
+        tips?: Hash[],
```

Comment:
tips should be `ReadonlyArray<Hash>`

Reply:
Fixed in https://github.com/iotaledger/iota.js/pull/317/commits/ef10d616c0dd1a9cc3a7438fb0348e8f7dbb9eea